.github/workflows/file-system-plugin-py-cd.yaml

```diff
@@ -0,0 +1,59 @@
+name: file-system-plugin-py-cd
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+    paths:
+      - "implementations/py/**"
+
+jobs:
+  cd:
+    name: file-system-plugin-py-cd
+    if: |
+      github.event.pull_request.merged &&
+      startsWith(github.event.pull_request.title, '[CD]') == true &&
```

Comment:
I think it would make sense if we change this to `release/py` (for python) & `release/rs` - wdyt?

Reply:
this is just the first iteration, we can improve this after if you'd like

Reply:
It's alright since it will only trigger for python release

Reply:
i dont think that's correct; we have the same string in the js cd workflow https://github.com/polywrap/file-system/blob/main/.github/workflows/file-system-plugin-js-cd.yaml#L16 (it's just that the condition is wrong lol)